plugins/rplanners/rplanners.h

```diff
@@ -305,7 +305,7 @@ class SpatialTree : public SpatialTreeBase
         RAVELOG_VERBOSE("computed in %fs", (1e-9*(utils::GetNanoPerformanceTime()-starttime)));
     }
 
-    virtual ExtendType Extend(const vector<dReal>& vTargetConfig, NodeBasePtr& lastnode, bool bOneStep=false)
+    virtual ExtendType Extend(const vector<dReal>& vTargetConfig, NodeBasePtr& lastnode, bool bOneStep=false, bool saveStatusData=false)
```

Comment:
Instead of exposing saveStatusData, why not expose `constraintFilterOptions` with default `0xffff|CFO_FillCheckedConfiguration`

Reply:
Or, `additionalConstraintFilterOptions`